index.html

```diff
@@ -53,6 +53,9 @@ <h1 id="anagram">YRGANEM</h1>
                         <div id="hintButton">
                             <p id="hint"></p>
                         </div>
+                        <button id="rescrumbleButton">
+                            Rescrumble
+                        </button>
```

Comment:
Other task is to create button. Please remove & make sure they are giving you the right id.